### ISSUE TITLE
Enable LDAP login via university ldap 

### ISSUE THREAD

**Issue by jdittrich** [BauhausUniversity/MedienWiki#23]
Remembering their login-credentials was a problem for students in usability-tests. Using the stable [LDAP-Extention](https://www.mediawiki.org/wiki/Extension:LDAP_Authentication) we might be able to allow them to use their normal login which is than verified centrally. 
(What do you think?)

**Comment by mxa** [comment]
+1 good idea.
i often see duplicated accounts where students just create a new login because the have forgot their original one.

m.

Am 05.06.2013 um 18:59 schrieb jdittrich notifications@github.com:

**Comment by audiocommander** [comment]
1. I once tried to connect our ㎎-server with the OpenDirectory system of the university and it was a horrible (horrible!) mess until I finally gave up after having fiddled around for two months. 
2. it's typically an all-or-nothing and we designed the Wiki to be usable by non-members of the Uni, so I don't think this is a good thing to do.
3. We don't fulfill any of the requirements (need to install additional libs on the server). We have no SSH access to the server and we cannot modify the PHP environment (or anything else).
   https://www.mediawiki.org/wiki/Extension:LDAP_Authentication/Requirements

So: unless someone can do it within our entitlements and volunteers to set it up, I'll close this issue.
